config/server/tce.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+const camelCase = require('lodash/camelCase');
+
+const envs = process.env;
+const regex = new RegExp('^TCE_(.*)');
+
+const tceConfig = Object.keys(envs)
+  .map(it => it.match(regex))
+  .filter(Boolean)
+  .map(([env, secret]) => ({ env, secret: camelCase(secret) }))
+  .reduce((config, { env, secret }) => ({
+    ...config,
+    [secret]: process.env[env]
+  }), {});
```

Comment:
Please consider if any of the following changes make it a bit easier to read:
```suggestion
const tceConfig = Object.keys(process.env)
  .map(key => key.match(/^TCE_(.*)/))
  .filter(Boolean)
  .reduce((config, [prefixedKey, key]) => ({
    ...config,
    [camelCase(key)]: process.env[prefixedKey]
  }), {});
```